packages/components/addon/components/hds/page-header/index.hbs

```diff
@@ -0,0 +1,30 @@
+{{!
+Copyright (c) HashiCorp, Inc.
+SPDX-License-Identifier: MPL-2.0
+}}
+<div class="hds-page-header" ...attributes>
+  {{yield (hash Breadcrumb=(component "hds/yield"))}}
+  <header class="hds-page-header__wrapper">
+    {{yield (hash IconTile=(component "hds/icon-tile" size="medium"))}}
+    <div class="hds-page-header__main">
+      <div class="hds-page-header__content">
+        <div class="hds-page-header__heading">
```

Comment:
I believe this was the only way to account for the badge coupled with the title/heading, while also justifying the actions to the end of the container. It's a little bit div heavy though, I'll admit.

Reply:
This was my recommendation, Melanie, to avoid nesting them, but if you think nesting the two landmarks is not a concern we can revert and apply the header element at the top level of this component.

Reply:
It's acceptable to nest landmark elements directly 👍